### DISCUSSION TITLE
How to get the current post/page inherit template slug using full site editor

### DISCUSSION THREAD

**Discussion by weweloo** [WordPress/gutenberg#54374] [Q&A]
Using the Gutenberg full site editor to create some template, how can I get which template’s slug is being used for the current page/post via php.

**Comment by ironprogrammer** [comment]
Hi, @weweloo 👋🏻 Try using [`get_page_template_slug`](https://developer.wordpress.org/reference/functions/get_page_template_slug/).

Closing this question for now, but please feel free to re-open if this didn't help.
